content/applications/inventory_and_mrp/barcode/setup/device_troubleshooting.rst

```diff
@@ -0,0 +1,214 @@
+
+   When scanning a barcode, the scanner might beep, indicating a successful scan, but there is no
+   data output in the app.
+
+   To fix this issue, adjust the output method to *keyboard analogue* in the *Scanner* app on the
+   device.
+
+   From the device's home screen, click :menuselection:`Scanner App --> Settings`. From the
+   :guilabel:`Settings` page, click :guilabel:`Output Mode`. The resulting pop-up window presents
+   the different output options available to users. Select :guilabel:`Keyboard Mode`, then click
+   :guilabel:`OK`.
+
+   .. image:: device_troubleshooting/device-troubleshooting-output-mode-popup.png
+      :align: center
+      :alt: Output mode pop-up window on MUNBYN scanner.
+
+   Go back to the app that needs to be scanned, and click on the :guilabel:`input dialog box` first
+   before scanning. Finally, perform a test scan to ensure the MUNBYN Android scanner is working as
+   intended.
+
+Datalogic Android devices
+-------------------------
+
+When using Datalogic Android scanners, ensure the following configurations are set to prevent
+errors.
+
+To view and configure all settings for the scanner, use the *Settings* app on the Datalogic Android
+device. From the applications menu, select :menuselection:`Settings --> System --> Scanner
+Settings`.
+
+From the resulting list of settings, select :guilabel:`Wedge`. From this menu, under the
+:guilabel:`Keyboard wedge` section, ensure that the :guilabel:`Enable keyboard wedge` feature is
+activated.
+
+Then, also under the :guilabel:`Keyboard wedge` section, locate the :guilabel:`Keyboard wedge input
+mode` option. By default, the input mode is set to :guilabel:`Text injection`.
+
+.. image:: device_troubleshooting/device-troubleshooting-wedge-menu.png
+   :align: center
+   :alt: Wedge configuration menu on Datalogic scanner.
+
+Click :guilabel:`Keyboard wedge input mode`, and change the setting to :guilabel:`Key pressure`.
+This ensures that scanned barcodes are translated into keyboard strokes, instead of being injected
+into the text area.
+
+.. image:: device_troubleshooting/device-troubleshooting-keyboard-wedge-input.png
+   :align: center
+   :alt: Keyboard wedge input mode selection on Datalogic scanner.
+
+Once all those steps have been taken, perform a test scan to ensure the Datalogic Android scanner is
+working as intended.
+
+.. _barcode/setup/screenless-scanners:
+
+Screenless scanners
+===================
+
+Screenless scanners are barcode scanning devices that have no screens. These include USB scanners
+and bluetooth scanners.
+
+.. important::
+   Odoo supports most USB and Bluetooth barcode scanners, as they all emulate a keyboard. However,
+   to verify that a scanner is compatible with a specific keyboard layout (or can be configured to
+   do so), refer to Odoo's `Inventory & Barcode Compatible Hardware documentation
+   <https://www.odoo.com/app/inventory-hardware>`_.
```

Comment:
Updating link text here too, I also prefer to keep "documentation" out of the link text :)
```suggestion
   do so), refer to Odoo's `Inventory & Barcode compatible hardware
   <https://www.odoo.com/app/inventory-hardware>`_ documentation.
```

Reply:
Suggestion implemented ✅